core/rawdb/database.go

```diff
@@ -213,12 +211,7 @@ func NewDatabaseWithFreezer(db ethdb.KeyValueStore, freezer string, namespace st
 
 	if ReadAncientType(db) == PruneFreezerType {
 		log.Warn("prune ancinet flag is set, may start fail, can add pruneancient parameter resolve")
-	}
-
-	// Create the idle freezer instance
-	frdb, err := newFreezer(freezer, namespace, readonly, freezerTableSize, FreezerNoSnappy)
-	if err != nil {
-		return nil, err
+		return nil, errors.New("pruneancient was set, please add pruneancient parameter")
```

Comment:
pruneancient => pruneAncient

Reply:
At present, the geth parameters are all lowercase. If we do other inline-prune features later, we can use a set of commands separately. we can use it like this for the time being.